src/RequestMatcher.php

```diff
@@ -164,6 +164,38 @@ public static function getSiteVariables($site = NULL) {
     return $vars;
   }
 
+  /**
+   * Gets the variant host.
+   *
+   * @param string $variant
+   *   Variant name.
+   *
+   * @return string|null
+   *   Variant url or null if variant incorrect.
+   */
+  public function getHostForSiteVariant($variant) {
+    if (!in_array($variant, $this->variants)) {
+      return NULL;
+    }
+    $site_variables = static::getSiteVariables();
+    $variant_host = $variant . $this->variantSeparator . $site_variables['SITE_HOST'];
```

Comment:
will produce wrong Host name if current site host is already a variant, see https://github.com/drunomics/multisite-request-matcher/
-> use MAIN host.